scenarios/barnoux/crs-anomaly-score.yaml

```diff
@@ -0,0 +1,15 @@
+type: trigger
+name: barnoux/crs-anomaly-score
+description: "Web exploitation detected via Core Rule Set inbound anomaly scoring set by the user in crs-setup.conf"
+filter: evt.Meta.log_type == 'modsecurity' && evt.Parsed.ruleid == '949110'
+blackhole: 2m
```

Comment:
If you have a blackhole key it be best to also have a groupby since a single trigger will blackhole the whole bucket

Reply:
is it ok now ?